### PR TITLE
istio: add install script

### PR DESCRIPTION
**Pull request by tomerlange**
This PR includes an install script which is the first step towards supporting istio environment on a kubernetes platform.
for testing, run the following commands from the path `scripts/ci/`:

    ./install-istio.sh install
    ./install-istio.sh start
    ./install-istio.sh stop
    ./install-istio.sh uninstall
    ./install-istio.sh status

### CODE REVIEW

File: scripts/ci/install-istio.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+install_istio() {
+        curl -L "$ISTIO_URL" | tar xz
+        if [ $? != 0 ]; then
+                echo "failed to download $url"
+                exit 1
+        fi
+
+        chmod -R 0777 $ISTIO_PKG
```
Comment (nplanel) [inline scripts/ci/install-istio.sh]: are you still using DOS ?
please change only the used binary to 755

**Comment by tomerlange** [comment]
The permission format is fixed.

File: scripts/ci/install-istio.sh (hunk 1 above)
Comment (tomerlange) [reply scripts/ci/install-istio.sh]: Fixed.

**Comment by tomerlange** [comment]
run tests

File: scripts/ci/install-istio.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+install_istio() {
+        curl -L "$ISTIO_URL" | tar xz
+        if [ $? != 0 ]; then
+                echo "failed to download $url"
+                exit 1
+        fi
+
+        chmod a+x $ISTIO_PKG
+        sudo cp $ISTIO_PKG/bin/istioctl $TARGET_DIR/.
+        sudo mv $ISTIO_PKG $TARGET_DIR/.
```
Comment (nplanel) [inline scripts/ci/install-istio.sh]: don't pollute /usr/bin
install the istio-1.0.1 directory to /usr/share/istio

File: scripts/ci/install-istio.sh [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+ISTIO_OBJECTS="$TARGET_DIR/$ISTIO_PKG/install/kubernetes/istio-demo.yaml"
+ISTIO_NS=istio-system
+
+uninstall_istio() {
+        sudo rm -rf $TARGET_DIR/$ISTIO_PKG
+        sudo rm -rf $TARGET_DIR/istioctl
+}
+
+install_istio() {
+        curl -L "$ISTIO_URL" | tar xz
```
Comment (nplanel) [inline scripts/ci/install-istio.sh]: insert
 istiodir=$(mktemp -d)
 cd $istiodir
 curl ....

File: scripts/ci/install-istio.sh (hunk 2 above)
Comment (nplanel) [inline scripts/ci/install-istio.sh]: add rm -rf $istiodir

**Comment by hunchback** [comment]
@tomerlange - please address @nplanel 's comments - I need this PR merged soon as my work depends on it. Thanks!

File: scripts/ci/install-istio.sh (hunk 2 above)
Comment (tomerlange) [reply scripts/ci/install-istio.sh]: Minikube and Kubectl are located in the same place; 
We want the installation be accessible by PATH...

**Comment by tomerlange** [comment]
run skydive-functional-tests-backend-orientdb

File: scripts/ci/install-istio.sh (hunk 3 above)
Comment (tomerlange) [reply scripts/ci/install-istio.sh]: done

File: scripts/ci/install-istio.sh (hunk 2 above)
Comment (tomerlange) [reply scripts/ci/install-istio.sh]: done

**Pull request merged**